### PR TITLE
Final touches micah

### PR DESCRIPTION
**Pull request by micah-wehrle**
This includes all of our last-minute improvements made to the project right before our presentation. Kerry and I already pulled in the master branch after the back-end implementation and so it should be ready to merge. Please take a look and make corrections as you are able, thanks.

Changes:
- Restructured navbar menu to be generated from a json object, instead of being hard-coded.
- Adjusted coloring of the weather alert timeout bar to indicate the severity of the alert

### CODE REVIEW

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -13,57 +13,52 @@ <h5 class="offcanvas-title" id="offcanvasNavbarLabel">Menu</h5>
...
-          </li>
-          <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-              Mini-projects
-            </a>
-            <ul class="dropdown-menu">
-              <li><a class="dropdown-item" routerLink="we-track">weTrack</a></li>
-              <li>
-                <hr class="dropdown-divider">
+          <!-- this should be its own components, it works but very messy -->
```
Comment (micah-wehrle) [inline src/app/components/header/header.component.html:16]: man you really should make this its own component!

Comment (micah-wehrle) [reply src/app/components/header/header.component.html:16]: done

Comment (StreamSG) [inline src/app/components/header/header.component.html:16]: That's what i said!

Comment (micah-wehrle) [reply src/app/components/header/header.component.html:16]: I thought you said "that's what she said"

**Review by micah-wehrle — COMMENTED** [review]
I think it's fine.

File: src/app/components/header/sidebar-list/sidebar-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import { MenuLink } from '../header.component';
+
+@Component({
+  selector: 'app-sidebar-list',
+  templateUrl: './sidebar-list.component.html',
+  styleUrls: ['./sidebar-list.component.scss']
+})
+export class SidebarListComponent implements OnInit {
+  @Input('menuLink') menuLink: MenuLink;
```
Comment (looksliketyler) [inline src/app/components/header/sidebar-list/sidebar-list.component.ts:11]: why assigning a value of `'menuLink'`?

Comment (micah-wehrle) [reply src/app/components/header/sidebar-list/sidebar-list.component.ts:11]: resolved, removed redundancy

**Comment by micah-wehrle** [comment]
@remindme delete branch in two weeks, around 5/3/2023

**Pull request merged**